### PR TITLE
traffic.sh - Traffic info

### PR DESCRIPTION
**Pull request by v-sens**
Gets repository traffic info.
(https://developer.github.com/v3/repos/traffic/)

### CODE REVIEW

File: traffic.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+source "base_api.sh"
+source "utils.sh"
+
+function traffic_clones() {
+  local -r TOKEN="$1"
+  local -r OWNER="$2"
+  local -r REPO="$3"
+  body=$(base_api::get "$TOKEN" "repos/$OWNER/$REPO/traffic/clones")
+  echo "$body" | jq -r '.count, .uniques'
+  }
```
Comment (MarounMaroun) [inline traffic.sh]: Can you please align the closing brackets, here and in other methods?

File: traffic.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  echo "$body" | jq -r '.[] | .referrer, .count, .uniques'
+  }
+
+function traffic_ref_path() {
+  local -r TOKEN="$1"
+  local -r OWNER="$2"
+  local -r REPO="$3"
+  body=$(base_api::get "$TOKEN" "repos/$OWNER/$REPO/traffic/popular/paths")
+  echo "$body" | jq -r '.[] | .path, .title, .count, .uniques'
+  # You can also replace it with:
```
Comment (MarounMaroun) [inline traffic.sh:28]: 👍

File: traffic.sh [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  # for more readable output.
+  }
+
+function traffic_views() {
+  local -r TOKEN="$1"
+  local -r OWNER="$2"
+  local -r REPO="$3"
+  body=$(base_api::get "$TOKEN" "repos/$OWNER/$REPO/traffic/views")
+  echo "$body" | jq -r '.count, .uniques'
+  }
```
Comment (MarounMaroun) [inline traffic.sh]: Better having a line break at the end of the file.

**Review by MarounMaroun — COMMENTED** [review]
Great addition! Thanks.

Just minor comments for the sake of the project's conventions.

File: traffic.sh (hunk 1 above)
Comment (v-sens) [reply traffic.sh]: Yes, of course, I’ll do it now

Comment (MarounMaroun) [reply traffic.sh]: @wardsenz Thanks!

**Pull request merged**